src/main/network/router/mod.rs

```diff
@@ -72,6 +72,12 @@ impl Router {
     }
 }
 
+impl Default for Router {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
We're in the middle of rewriting the `Router` and related networking modules, so I think we should avoid implementing `Default` until we know it doesn't require parameters to initialize.